### DISCUSSION TITLE
ManufacturerName and ModelIdentifier not received by Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by Caroline-Bakkene** [Koenkk/zigbee2mqtt#9840] [Devices]
Hello!

I have an occupancy sensor connected to a LPSTK-CC1352R (used as an end-device). I am using zigbee2mqtt (with mosquitto as the broker) on a RPi, and a CC2652R as a coordinator.

In the zigbee2mqtt terminal on the Pi, I can see the status of the occupancy (0 or 1), and that my device has joined the network. My problem is that it seems like it can not read the ManufacturerName or ModelIdentifier, as it says: "Received message from unsupported device with Zigbee model ' ' and manufacturer name ' '."

I also have a occupancy sensor connected to a CC2652R (ZED), and it works perfectly for that one. I have compared header files and zcl_config files and they look identical. The necessary clusters/attributed has been added, the external converter, in addition to these code lines (used exactly the same for both boards):
![image](https://user-images.githubusercontent.com/75332596/143451237-a05ab8cd-7d8f-467e-8bcb-037d4eab4c69.png)

When first connecting/joining the network, it seems like it is not performing the interviewing. In the frontend, it says that the interview is completed, but this was not shown in the debug log. Have tried to delete the device and connect it several times, but same issue every time. 

Any ideas of why it is not working on my CC1352R board? Is there a possibility that my device is not interviewed?

Thanks!

**Comment by Koenkk** [comment]
During interviewing z2m should send read requests for both the modelid and manufacturer name, you can debug what your device responds with sniffing: https://www.zigbee2mqtt.io/advanced/zigbee/04_sniff_zigbee_traffic.html#with-cc2531

**Reply by Caroline-Bakkene** [reply]
Thanks for the reply. Have used sniffing now, and it confirms what I wrote earlier. When the device is first connected, it is not interviewed. What can be the cause of that? Is there any settings that can have disabled interviewing?

**Reply by Koenkk** [reply]
You can see this in the herdsman debug logging:

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.
